File: BookHubWeb/BookHubAPI/Models/Event.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BookHubAPI.Models
{
    [Table("Events")]
    public class Event
    {
        [Key]
        [Column("event_id")]
        public int EventId { get; set; }

        [Required, MaxLength(255)]
        [Column("title")]
        public string Title { get; set; } = string.Empty;

        [Column("description")]
        public string? Description { get; set; }

        [Column("start_date")]
        public DateTime? StartDate { get; set; }

        [Column("end_date")]
        public DateTime? EndDate { get; set; }

        [Column("image_banner_url")]
        public string? ImageBannerUrl { get; set; }

        [Column("is_active")]
        public bool IsActive { get; set; } = true;
    }
}

File: AppBookHub _NhomLapGhep/app/src/main/java/com/example/bookhub/models/Event.java
package com.example.bookhub.models;

public class Event {
    private int id; // Đã có biến này
    private String title;
    private String description;
    private String startDate;
    private String endDate;
    private String imageUrl;
    public int getId() { return id; }

    public String getTitle() { return title; }
    public String getDescription() { return description; }
    public String getStartDate() { return startDate; }
    public String getEndDate() { return endDate; }
    public String getImageUrl() { return imageUrl; }
}